.eslintrc.js

```diff
@@ -8,13 +8,19 @@ module.exports = {
     node: true
   },
   extends: [require.resolve('./lib/configs/recommended'), 'plugin:eslint-plugin/all'],
-  plugins: ['eslint-plugin'],
+  plugins: ['eslint-plugin', 'no-only-tests'],
```

Comment:
This change adds the rule to the ESLint config of this project. We should add it to some configs we provide as well. Can you make sure to include the rule in `lib/configs/recommended.js` as part of this pull request?

Reply:
Of course :facepalm:
Indeed I've removed it from this project ESLint config, because tests here use `ruleTester.run` which won't get advantage of the new no-only-tests rule.